sources/Context.cpp

```diff
@@ -314,11 +317,12 @@ Context::execute() {
     simu->clean();
     break;
   }
-  case dfl::inputs::Configuration::SimulationKind::SECURITY_ANALYSIS:
+  default: {
```

Comment:
Please use a case ... to avoid silent behaviors if we decide to add an item in the enum